optimum/exporters/tasks.py

```diff
@@ -177,6 +177,7 @@ class TasksManager:
             "question-answering": "AutoModelForQuestionAnswering",
             "semantic-segmentation": "AutoModelForSemanticSegmentation",
             "text-to-audio": "AutoModelForTextToSpectrogram",
+            "text-to-waveform": "AutoModelForTextToWaveform",
```

Comment:
Shouldn't it be text-to-audio?

https://huggingface.co/datasets/huggingface/transformers-metadata/blob/main/pipeline_tags.json#L983